Content.Server/GameTicking/GameTicker.Spawning.cs

```diff
@@ -256,7 +257,7 @@ public void MakeObserve(IPlayerSession player)
             newMind.ChangeOwningPlayer(data.UserId);
             newMind.AddRole(new ObserverRole(newMind));
 
-            var mob = SpawnObserverMob();
+            var mob = SpawnObserverMob(_adminManager.IsAdmin(player));
```

Comment:
This needs to check if they can run the `aghost` command, not just if they're admins. For example codermins on Wizard's Den can't aghost (iirc) but are admins.